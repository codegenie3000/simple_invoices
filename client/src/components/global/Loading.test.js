import React from 'react';
import { shallow } from 'enzyme';

import Loading from './Loading';

describe('<Loading/>', () => {
    it('tests if it renders correctly', () => {
        const component = shallow(<Loading/>);
        expect(component).toMatchSnapshot();
    });
});