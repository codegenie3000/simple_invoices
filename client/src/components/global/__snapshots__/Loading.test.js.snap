// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> tests if it renders correctly 1`] = `
<div>
  loading...
</div>
`;
